frontend/src/components/SupportingDocumentUpload.vue

```diff
@@ -498,9 +496,12 @@ export default {
       this.isLoading = true;
       try {
         await this.getDocuments(this.applicationId);
-        this.uploadedSupportingDocuments = this.uploadedDocuments.filter(
-          (document) => this.navBarList.findIndex((item) => item.facilityId == document.ccof_facility) > -1,
-        );
+        this.uploadedSupportingDocuments = this.uploadedDocuments.filter((document) => {
+          return (
+            this.navBarList.findIndex((item) => item.facilityId == document.ccof_facility) > -1 &&
```

Comment:
This should be === unless the types are known to be different.

Reply:
done - also updated the rest of the file